cql3/statements/schema_altering_statement.cc

```diff
@@ -64,7 +64,7 @@ schema_altering_statement::execute0(query_processor& qp, service::query_state& s
     auto& mm = qp.get_migration_manager();
     ::shared_ptr<cql_transport::event::schema_change> ce;
 
-    if (mm.is_raft_enabled() && this_shard_id() != 0) {
+    if (this_shard_id() != 0) {
```

Comment:
This, as well as the test changes, should go in independently of the rest of the patch set. I believe Gleb didn't do it to have zero impact on existing install base while raft is still experimental.

Reply:
Yes. I did not to affect the raft disabled code works. Not sure what impact it may have, but better to be safe than sorry. @kbr- Why do you want to unconditionally bonce to shard 0 now?

Reply:
1. This is the state we're aiming to eventually achieve
2. I remove `migration_manager::is_raft_enabled()` later in the PR. Now deciding to use or not use group 0 for operations is done by inspecting `group0_guard::with_raft()`, which is based on where in the upgrade procedure we're at.